packages/Overlay/src/index.js

```diff
@@ -1,3 +1 @@
 export { default } from "./Overlay";
-export { default as Backdrop } from "./Backdrop";
-export { default as Wrapper } from "./Wrapper";
```

Comment:
This is a breaking change to the API 💥 
No Paprika components were consuming these however, and it seems unlikely that our apps would be.